src/main/php/test/execution/TestClass.class.php

```diff
@@ -24,6 +24,9 @@ public function __construct($arg, $selection= null) {
   /** @return string */
   public function name() { return $this->context->type->name(); }
 
+  /** @return ?string */
+  public function declaringFile() { return $this->context->type->class()->reflect()->getFileName(); }
```

Comment:
Should accessing the file name be part of the https://github.com/xp-framework/reflection API?